module1/homework/make_unique/make_unique-ut.cpp

```diff
@@ -44,12 +47,27 @@ TEST(make_unique, shouldMakeUniqueToEmpytyMyTestTypeObject) {
     std::unique_ptr<MyTestType> uniqMyTestType{new MyTestType{}};
     auto testUniquePtr = cs::make_unique<MyTestType>();
     ASSERT_EQ(*uniqMyTestType, *testUniquePtr);
+    ASSERT_EQ(testUniquePtr->intValue_, intValue);
+    ASSERT_EQ(testUniquePtr->doubleValue_, doubleValue);
+    ASSERT_EQ(testUniquePtr->boolValue_, boolValue);
 }
 
-TEST(make_unique, shouldMakeUniqueToMyTestTypeObject) {
-    std::unique_ptr<MyTestType> uniqMyTestType{new MyTestType{2, 2.5, 0}};
-    auto testUniquePtr = cs::make_unique<MyTestType>(2, 2.5, 0);
-    ASSERT_EQ(*uniqMyTestType, *testUniquePtr);
+TEST(make_unique, shouldMakeUniqueToMyTestTypeObjectWithIntLValue) {
+    int intLValue{1};
+
+    auto testUniquePtr = cs::make_unique<MyTestType>(intLValue, 1.5, true);
+    ASSERT_EQ(testUniquePtr->intValue_, intLValue);
+    ASSERT_EQ(testUniquePtr->doubleValue_, doubleValue);
+    ASSERT_EQ(testUniquePtr->boolValue_, boolValue);
+}
+
+TEST(make_unique, shouldMakeUniqueToMyTestTypeObjectWithDoubleLValue) {
+    double doubleLValue{1.5};
+
+    auto testUniquePtr = cs::make_unique<MyTestType>(1, doubleLValue, true);
+    ASSERT_EQ(testUniquePtr->intValue_, intValue);
+    ASSERT_EQ(testUniquePtr->doubleValue_, doubleLValue);
+    ASSERT_EQ(testUniquePtr->boolValue_, boolValue);
```

Comment:
it's not the case. Still - whatever you pass it will be in your structure. 
You can eg. set intValue_ to 0 when it is passed as lvalue and to 1 if it is rvalue. The same for double. Then just check if values are proper. It means that a proper constructor was called.